docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: '2'
+
+services:
+  compile:
+    build:
+      context: ${PWD}/docker
+      dockerfile: Dockerfile.compile
+    environment:
+      - CXX
+    volumes:
+      - build:/build
+      - ${TRAVIS_BUILD_DIR}:/source
+    working_dir: /build
+  db:
+    environment:
```

Comment:
On Necronia I made it this way to wait till build container is eventualy gone : 
1. Linked server container with build container, then added this code to entry point script

###########################
##Check if server is build, is so postpone start up
###########################
BUILD_CONTAINER=necronia-build

ping -c 1 $BUILD_CONTAINER;
while [ "$?" -eq 0 ]; do echo 'Server is still building...'; sleep 15; ping -c 1 $BUILD_CONTAINER; done;